src/components/ColorPicker.re

```diff
@@ -0,0 +1,34 @@
+module Styles = {
+  open Css;
+
+  let field =
+    style([
+      alignItems(center),
+      display(flexBox)
+    ]);
+
+  let input =
+    style([
+      borderWidth(px(0)),
+      background(none),
+      cursor(`pointer),
+      height(px(24)),
+      width(px(20)),
+      unsafe("-webkit-appearance", "none")
+    ]);
+
+  let label =
+    style([
+      paddingLeft(px(Theme.Spacing.baseHalf))
+    ]);
+};
+
+[@react.component]
+let make = (~className="", ~label="") => {
+   <div className=(className ++ " " ++ Styles.field)>
```

Comment:
as we are starting to use this a lot maybe it's time to use https://www.npmjs.com/package/re-classnames

Reply:
ok, I will pack this implementation in a specific PR, what do you think?

Reply:
Does this className comes from bs-css? Why not use `Css.(merge([Styles.field, className]))`?

Reply:
It doesn't! It's just because it has more functionality, even to deal with option values